src/pages/K8s.vue

```diff
@@ -141,16 +141,17 @@ export default {
   data() {
     return {
       /** @type {{ key: string, message: string, level: string }} */
-      notifications:        { },
-      state:                ipcRenderer.sendSync('k8s-state'),
-      currentPort:          0,
-      currentEngine:        ContainerEngine.NONE,
-      containerEngineNames: ContainerEngineNames,
+      notifications:         { },
+      state:                 ipcRenderer.sendSync('k8s-state'),
+      currentPort:           0,
+      currentEngine:         ContainerEngine.NONE,
+      containerEngineNames:  ContainerEngineNames,
       /** @type Settings */
-      settings:             defaultSettings,
+      settings:              defaultSettings,
       /** @type {import('@/k8s-engine/k8s').VersionEntry[] */
-      versions:             [],
-      progress:             {
+      versions:              [],
+      versionListRestricted: false,
```

Comment:
As elsewhere — please describe what "restricted" means. Or just call this `cachedVersionsOnly`.

Reply:
ok